historical-adherence-query/config.yml

```diff
@@ -0,0 +1,97 @@
+---
+title: Retrieving Notifications from a Historical Adherence Query
+description: Tutorial on how to subscribe to a historical adherence notifications and retrieve the results of a query
+categories:
+- Notifications
+steps:
+- title: Introduction
+  content: |
+    This tutorial shows how to subscribe to historical adherence notifications for a query and retrieve the resulting downloadUrls.
+    Subscribing to notifications is how you are able to retrieve events that include the results of your requests.
+    With historical adherence queries specifically, after your intital request your results will begin processing.
```

Comment:
spelling (intital)

You should probably run this whole change set through spell checker if you haven't already.